source/devs/forge/contributing.rst

```diff
@@ -37,27 +37,32 @@ You can download Eclipse here: https://eclipse.org/downloads/
 
 You can download IntelliJ IDEA here: https://www.jetbrains.com/idea/download/
 (Choose the "Community Version" unless you want to pay for the Ultimate Edition.
-You do not need the Ultimate Edition to develop plugins, as the Community Version still contains all the tools you need.)
+You do not need the Ultimate Edition, as the Community Version still contains all the tools you need.)
 
-git Download
+Git Download
 ~~~~~~~~~~~~
 
-The download page for git is here:  http://git-scm.com/downloads
+The download page for Git is here:  http://git-scm.com/downloads
 
 Make sure to install the appropriate version for your operating system.
 
-Linux users, git will be available on your distro's package manager, usually under the name "git".
+For Linux users, Git will be available on the distro's package manager, usually under the name "git".
 
 Install it like any other application, and reboot.
 
-Running Gradle
---------------
+Workspace Setup
+---------------
 
-Once these are downloaded, installed, and you've rebooted (again), then you can begin to setup the workspace specific for Sponge.
+Once these are downloaded, installed, and you've rebooted (again), 
+then you can begin to setup the workspace specific for Sponge.
 
-(We're going to assume that your `PATH` is correct as well, each operating system has its own way of specifying new directories to search for commands. In Unix-based systems the environment variable is `$PATH`.  In Windows systems it is `%PATH%`%. It's advised to construct your profile in your system such that when you login to the computer, the `PATH` is setup correctly vs. manually updating the `PATH` before working with Sponge)
+(We're going to assume that your `PATH` is correct. 
+Each operating system has its own way of specifying new directories to search for commands. 
+In Unix-based systems the environment variable is `$PATH`.  In Windows systems it is `%PATH%`%. 
```

Comment:
I have no idea how `.rst` files are rendered, but it should display `%PATH%` as an end result. Again I have no idea if it will be.